styles.css

```diff
@@ -0,0 +1,366 @@
+
+
+h1 {
+  font-family: 'Bitter', serif;
+  color: #093C60;
+}
+
+h2 {
+  font-family: 'Indie Flower', cursive;
+  color: #81B742;
+}
+
+h3 {
+  font-family: 'Bitter', sans-serif;
+  color: #093C60;
+}
+
+h3.footer-h3 {
+  color: #ECECEC;
+  font-family: 'Muli', sans-serif;
+  line-height: 1.5;
+}
+
+h4 {
+  font-family: 'Bitter', sans-serif;
+  color: #093C60;
+  line-height: 0.5;
+}
+
+p {
+  color: #373943;
+  font-family: 'Muli', sans-serif;
+  line-height: 1.5;
+}
+
+p.small {
+  color: #373943;
+  font-family: 'Muli', sans-serif;
+  line-height: 1.5;
+  font-size: 9pt;
+}
+
+p.light{
+  color: #ECECEC;
+  text-align: right;
+}
+
+p.footer-p {
+  color: #ECECEC;
+  font-family: 'Muli', sans-serif;
+  line-height: 1.5;
+}
+
+a {
+  color: inherit;
+  font-family: inherit;
+  text-decoration: none;
+  margin: -10px -10px -10px -10px;
+  padding: 10px 10px 10px 10px;
+}
+
+a.nav {
+  color: inherit;
+  font-family: inherit;
+  text-decoration: none;
+  margin: -10px -10px -10px -10px;
+  padding: 10px 10px 10px 10px;
+}
+
+a.nav:hover {
+  color: #81B742;
+}
+
+a.footer-nav {
+  color: #ECECEC;
+  font-family: 'Muli', sans-serif;
+  text-decoration: none;
+  margin: -10px -10px -10px -10px;
+  padding: 10px 10px 10px 10px;
+}
+
+a.footer-nav:hover {
+  color: #81B742;
+}
+
+.container {
+  display: grid;
+}
+
+header {
+  background-image: url(images/header-image.jpg);
+  padding: 1em 1em 7em 4em;
+  margin: -1em -1em 0em -1em;
+}
+
+nav {
+  background-color: #ECECEC;
+  grid-column: 1 / span 8;
+  grid-row: 2 / span 1;
+  padding: .25em .25em .25em 10em;
+  margin: 0em -1em 0em -1em;
+}
+
+nav.footer-nav {
+  background-color: #093C60;
+  padding: 0em;
+  margin: 2em 0em 2em 0em;
+}
+
+article.blog-article {
+  padding-bottom: 3em;
+}
+
+.header-flex {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  padding-bottom: 2em;
+}
+
+div.header-box {
+  padding: 1em 1em 1em 1em;
+  margin-bottom: 1em;
+  margin-right: 1em;
+}
+
+.dog-flex {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+div.dog-box {
+  width: 12em;
```

Comment:
You'll want to make this some sort of percentage based value so it grows and shrinks a bit when the browser window changes widths.